span.go

```diff
@@ -58,3 +62,30 @@ type Span interface {
 	// TraceContext.
 	AddToGoContext(goCtx context.Context) (Span, context.Context)
 }
+
+// See Span.Log(). Every LogData instance should specify at least one of Event
+// and/or Payload.
+type LogData struct {
+	// The timestamp of the log record; if set to the default value (the unix
+	// epoch), implementations should use time.Now() implicitly.
+	Timestamp time.Time
+
+	// Event (if non-empty) should be the stable name of some notable moment in
```

Comment:
I mean if it's not set, basically, yeah. Go makes it extremely annoying to pass pointers to literal strings... you would want to write

```
  ...
  Event: &"my_literal_string",
  ...
```

, but that doesn't compile. This does, but is super gross:

```
func strPtr(s string) { return &s }

  ...
  Event: strPtr("my_literal_string"),
  ...
```

gah.